chains/substrate/listener/listener.go

```diff
@@ -0,0 +1,90 @@
+package listener
+
+import (
+	"math/big"
+	"time"
+
+	"github.com/ChainSafe/chainbridge-core/relayer/message"
+	"github.com/ChainSafe/sygma-relayer/chains/substrate/events"
+
+	"github.com/ChainSafe/chainbridge-core/store"
+	"github.com/centrifuge/go-substrate-rpc-client/types"
+	"github.com/rs/zerolog/log"
+)
+
+var BlockRetryInterval = time.Second * 5
```

Comment:
It would probably make sense to add this to config and have 5 seconds as the default value.